frontend/public/locales/en/gcweb.json

```diff
@@ -47,6 +47,14 @@
     "option-02": "Return to the <home>home page</home>"
   },
   "input-label": {
-    "required": "required"
+    "error": "Error",
+    "required": "required",
+    "button": {
+      "submit": "Submit",
+      "cancel": "Cancel",
+      "change": "Change",
+      "confirm": "Confirm",
+      "edit": "Edit"
+    }
```

Comment:
I don't believe these belong in this file.

Reply:
This key is specific for the `InputLabel` GCWeb's component that I've created. No more keys should be added to `"input-label"`.

```json
"input-label": {
  "required": "required"
 }
```